gasometer/src/costs.rs

```diff
@@ -245,6 +245,14 @@ pub fn sstore_cost(
 	)
 }
 
+pub fn tload_cost(config: &Config) -> Result<u64, ExitError> {
+	Ok(config.gas_sload)
```

Comment:
```suggestion
	Ok(config.gas_storage_read_warm)
```

According to EIP 1153, the cost of TLOAD should be like a wrm SLOAD, so the correct value is `gas_storage_read_warm`.
It might not be equal to gas_sload in the future

Reply:
Fixed